src/injector.js

```diff
@@ -159,6 +119,47 @@ export default class Injector {
     }
   }
 
+  injectIntoDOM(dependencies, idx = 0, type = null) {
+    if (idx >= dependencies.length) { return; }
+
+    // inject order: explicitly provided < cached in local storage < printed
+    // raw only as fallback if printed fails
+    type = (dependencies[idx][type] && type) || (dependencies[idx]['cached'] && 'cached') || (dependencies[idx]['printed'] && 'printed');
```

Comment:
Hm. Isn't the `dependencies[idx]['cached'] && 'cached'` `"cached"` part always true?